sda-commons-server-kafka/README.md

```diff
@@ -363,6 +363,23 @@ kafka:
       pollInterval: 200
 ```
 
+You can disable the  health check manually if kafka is not essential for the functionality of your service,
+e.g. it's only used to invalidate a cache, notify about updates or other minor tasks that could fail without
+affecting the service so much it's no longer providing a viable functionality.
```

Comment:
kafka or Kafka?

Reply:
Fixed the inconsistent spelling, should be Kafka everywhere.